Makefile

```diff
@@ -28,15 +28,15 @@ spec:
 		bundle exec rspec
 
 lint_yaml:
-	$(YAMLLINT) -c yamllint.yml .
+	find . -name '*.yml' | grep -v -e '^\./vendor' | xargs $(YAMLLINT) -c yamllint.yml
```

Comment:
Is this a good excuse?

![screenshot from 2016-04-14 17 05 56](https://cloud.githubusercontent.com/assets/280032/14534936/41389b16-0263-11e6-816c-69a061afbd9c.png)